scripts/annotate-vendored-checkov/main_test.go

```
@@ -0,0 +1,180 @@
+	tests := []struct {
+		name            string
+		bundle          string
+		clusterRoleName string
+		deploymentName  string
+	}{
+		{name: "CDI", bundle: "cdi", clusterRoleName: "cdi-operator-cluster", deploymentName: "cdi-operator"},
+		{name: "KubeVirt", bundle: "kubevirt", clusterRoleName: "kubevirt-operator", deploymentName: "virt-operator"},
+	}
+
+	for _, test := range tests {
+		test := test
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "test" can be deleted (Go 1.22+)

(copyloopvar)